src/xo-app/home/index.js

```diff
@@ -205,6 +207,7 @@ const TYPES = {
 const DEFAULT_TYPE = 'VM'
 
 @addSubscriptions({
+  resourceSets: subscribeResourceSets,
```

Comment:
These subscriptions are only necessary when there are no objects to display.

Create a dedicated component which have the subscriptions and is used by `Home` when there are no objects.